libraries/botbuilder-dialogs/src/dialogContext.ts

```diff
@@ -1,90 +1,92 @@
-// tslint:disable
 /**
  * @module botbuilder-dialogs
  */
 /**
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { TurnContext, Activity } from 'botbuilder-core';
-import { DialogInstance, DialogTurnResult, DialogTurnStatus, DialogReason } from './dialog';
+import { Activity, TurnContext } from 'botbuilder-core';
+import { Choice } from './choices';
+import { Dialog, DialogInstance, DialogReason, DialogTurnResult, DialogTurnStatus } from './dialog';
 import { DialogSet } from './dialogSet';
 import { PromptOptions } from './prompts';
-import { Choice } from './choices';
 
-export interface DialogState
-{
+export interface DialogState {
     dialogStack: DialogInstance[];
 }
 
 /**
  * A context object used to manipulate a dialog stack.
- * 
+ *
  * @remarks
- * This is typically created through a call to `DialogSet.createContext()` and is then passed 
+ * This is typically created through a call to `DialogSet.createContext()` and is then passed
  * through to all of the bots dialogs and waterfall steps.
- * 
+ *
  * ```JavaScript
  * const conversation = conversationState.get(context);
- * const dc = dialogs.createContext(context, conversation);  
+ * const dc = dialogs.createContext(context, conversation);
```

Comment:
I really want you to change this comment as it's out of date; but that's out of scope for this PR... Do we have an issue for out of date doc strings?

Reply:
yes, I've started that in #396.  I can update this one though!

Reply:
I updated this one and all the other references to createContext I could find.